packages/nimble-components/src/table-column/date-text/tests/table-column-date-text.stories.ts

```diff
@@ -265,7 +266,9 @@ export const dateTextColumn: StoryObj<TextColumnTableArgs> = {
         customHour12: {
             name: 'custom-hour12',
             description:
-                'Refer to the option [documentation for the `Intl.DateTimeFormat()` constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat).'
+                'Refer to the option [documentation for the `Intl.DateTimeFormat()` constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat).',
```

Comment:
Do we still want to just re-direct users to this documentation, as our API is now misaligned?

Reply:
I've added a note about how our values map to the ones in the linked documentation.